test/sequential/test-debugger-launch.mjs

```diff
@@ -0,0 +1,35 @@
+import { skipIfInspectorDisabled } from '../common/index.mjs';
+skipIfInspectorDisabled();
+
+import { path } from '../common/fixtures.mjs';
+import startCLI from '../common/debugger.js';
+
+import assert from 'assert';
+const script = path('debugger', 'three-lines.js');
```

Comment:
Optional nit:

```suggestion
import assert from 'assert';

const script = path('debugger', 'three-lines.js');
```

(Separate `import` statements from the rest of the file.)

Reply:
Generally 2 empty lines